tests/helpers.py

```diff
@@ -88,7 +88,9 @@ def wait_for_resource_state(resource, state):
     return True
 
 
-def create_pod(interface_type=None, pvc_name=None, desired_status=constants.STATUS_RUNNING, wait=True):
+def create_pod(
+        interface_type=None, pvc_name=None, desired_status=constants.STATUS_RUNNING, wait=True, namespace=None
```

Comment:
fix indentation

Reply:
Also, this line looks longer than 80 chars - please fix

Reply:
Fixed.